preprocessors/charts.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/charts.schema.json",
+    "type": "object",
+    "title": "Charts preprocessor",
+    "description": "Extracts additional information from the datapoints present in the graph",
+    "properties": {
+        "dataPoints": {
+            "type": "array",
+            "title": "dataPoints",
+            "description": "The dataPoints in the graph",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "x": {
+                        "type": "number",
+                        "title": "x",
+                        "description": "x axis value"
+                    },
+                    "y": {
+                        "type": "number",
+                        "title": "y",
+                        "description": "y axis value"
+                    },
+                    "lowerPointsOnLeft": {
+                        "type": "number",
+                        "title": "Lower points on left",
+                        "description": "contiguous points lower than the data Point, lying on the left side on the x axis"
```

Comment:
This mention of "contiguous" is odd in context with the name and title not specifying any kind of range. This is the count of points to the left of the current point before a point is reached with an equal or higher value? Is this count actually useful if points aren't equally spaced on the x axis? Identical questions and concerns for the other higher/lower points on left/right keys.

Reply:
More questions...what does "lying on the left side on the x axis" mean here? I've been assuming left of the point but this isn't actually stated anywhere. Is this to the left of where the y axis was drawn on the screen? Is left/right defined some other way?